src/debugger/Debugger.h

```diff
@@ -180,12 +157,11 @@ class Debugger : public gc {
     void enableDebugger(Context* context);
     void disableDebugger();
 
-    virtual void init(const char* options, Context* context) = 0;
     virtual bool processEvents(ExecutionState* state, Optional<ByteCodeBlock*> byteCodeBlock, bool isBlockingRequest = true) = 0;
 
     uint32_t m_delay;
     ExecutionState* m_stopState;
-    Vector<CodeBlockData, GCUtil::gc_malloc_allocator<CodeBlockData>> m_codeBlockData;
+    std::vector<DebuggerOperationsRef::BreakpointLocationsInfo*> m_breakpointLocationsVector;
```

Comment:
Unfortunately, our GC cannot trace `std::vector` correctly.
If `m_breakpointLocationsVector` vector contains GC elements, than GCVector(`Vector`) should be used instead.

Reply:
This is a part where I am not sure what data structure would be the best choice. That vector should contain no pointers to Escargot internal data, and ideally its ownership could be passed to the API user. If the user don't needs it (e.g. it copies the data), we should just free it. How would you define such a structure?

Reply:
If `BreakpointLocationsInfo` is not a GC object and should not be tracked by garbage collector, then using std::vector has no problem. We need to free it correctly thats all.

Reply:
Currently the destructors should free it automatically.